python/ray/tests/kuberay/test_kuberay_node_provider.py

```diff
@@ -221,7 +257,7 @@ def mock_get(node_provider, path):
             [
                 {
                     "op": "replace",
```

Comment:
Add "tpu-group" to the `ScaleRequest`?

Reply:
The logic for `_submit_scale_request` isn't changed at all in this PR, the path for the tpu-group case will be the same as small-group and gpu-group.